### PR TITLE
fix: multiple pics for same user on map

### PR DESCRIPTION
**Pull request by brijeshsoni1300**
## Fixes Issue
Closes #31 

## Changes proposed

removed duplicate entries from events-list for Map only 
so there will be single pic for all entries of same user on Map

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EddieHubLive/pull/37"></a>

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function App() {
         ))}
       </div>
       <div className="w-2/3">
-        <Map events={events} />
+      <Map events={[...new Set(events)]} />
```
[the remark this answers is not in the document]
Comment (brijeshsoni1300) [reply src/App.js]: now its right

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Review by FahimFBA — APPROVED** [review]

**Comment by FahimFBA** [comment]
Looks good to me 👍

**Review by jaynarayan-vaishnav — COMMENTED** [review]
LGTM🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by eddiejaoude** [comment]
testing events

**Comment by eddiejaoude** [comment]
2nd test

**Comment by eddiejaoude** [comment]
3rd test

**Comment by eddiejaoude** [comment]
Oddly 3 images are still appearing on the map 🤔 

Screenshot 2022-07-11 at 19 49 45

**Pull request merged**